### PR TITLE
Package the helm distribution as well.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Trivial extension. Currently setting up a local concourse for test.

### CODE REVIEW

File: cf/tasks/package-bits.sh [hunk 1]
```diff
@@ -30,8 +30,9 @@ do
     mkdir tmp
 
     # kube configs
-    # helm charts
     unzip s3.kube-dist/scf-kube-*.zip -d tmp
+    # helm charts
+    unzip s3.helm-dist/scf-helm-*.zip -d tmp
```
Comment (mook-as) [inline cf/tasks/package-bits.sh]: We may need to extract them to `tmp/kube` and `tmp/helm`

Comment (andreas-kupries) [inline cf/tasks/package-bits.sh]: Hm. IIRC helm has a `helm` sub dir, and the other two `cf` and `uaa`. Should have no conflict.
Regardless, I will check what the concourse gives me.

Comment (andreas-kupries) [inline cf/tasks/package-bits.sh]: Seems that you are correct. The helm stuff is under `cf` as well, so it mixes with the kube cf.
Now fixing.

Comment (andreas-kupries) [inline cf/tasks/package-bits.sh]: Done with commit fa59247b7de0d90585ed351095e43e3fe59cd34a

Comment (andreas-kupries) [inline cf/tasks/package-bits.sh]: Please run this on your concourse and send me the resulting zip archive.
(On my local one it fails to upload to your minio, access denied)
(With a zip file in hand I should be able to complete the docs)

**Pull request merged**